### DISCUSSION TITLE
[Help Wanted] Does Standard Notes support Backrgound App Refresh on iOs?

### DISCUSSION THREAD

**Discussion by vectro** [standardnotes/forum#2742] [Suggestions]
Hi folks,

I am just wondering if the iOs Standard Notes app will sync notes in the background, or if the app has to be opened in order to sync.

I am thinking of a use case like a grocery list, where one might not have Internet access at the grocery store and would want to ensure that the latest list is always available.

Thanks,

Ian

**Comment by atmoio** [comment]
We do not make use of background refresh currently. But definitely something to keep in mind.
